readme.md

```diff
@@ -41,7 +41,7 @@ $ yarn add telegraf
 ### Examples
   
 ```js
-const Telegraf = require('telegraf')
+const { Telegraf } = require('telegraf')
```

Comment:
I think with `export {Telegraf as default}` introduced in e3d3589196587ff88a4d8231bfe268fcb4e21dce we can still recommend usage as `const Telegraf = require('telegraf')`. Or do I miss something?

Reply:
Actual CommonJS eqivalent is weirdly looking `const Telegraf = require('telegraf').default`. 

```js
const Telegraf = require('telegraf')
new Telegraf(token)
```
runs correctly, but makes TS complain:
```
This expression is not constructable.
  Type 'typeof import("telegraf/typings/index")' has no construct signatures.ts(2351)
```

This is exactly what I was trying to prevent with my overly complex `index.d.ts`.

Reply:
If it runs correctly for JavaScript users I would continue to suggest it that way.
For TypeScript users I wouldn't suggest using require as import from is the way to go. 

But im not sure what VisualStudio Code autocomplete does when the simple require() is used in JavaScript.

If we change the overall usage towards `const {Telegraf} = require('telegraf')` or `import {Telegraf} from 'telegraf'` then the default import can be removed...

I think the main goal should be to have as little changes to JavaScript users (while autocomplete and stuff like that works) as possible. The second goal should be to have TypeScript types which are reflecting as close as possible what the JavaScript code does. If the types are not working `as any` is a common way to handle telegraf currently anyway (Markup and Extra for example) as it falls back to the stable working JavaScript code.

Reply:
> But im not sure what VisualStudio Code autocomplete does when the simple require() is used in JavaScript.

It sees it as a namespace, but not a class.

To make it see it as a class as well, `export=` has to be used, but then no other exports are allowed, which forced me to awkwardly squeeze them into otherwise unneccesary namespace.